common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -297,6 +298,9 @@ def populate_course_fixture(self, course_fixture):
                         XBlockFixtureDesc(
                             'problem', self.beta_text, data=problem_data, metadata={"group_access": {0: [1]}}
                         ),
+                        XBlockFixtureDesc(
+                            'problem', self.audit_text, data=problem_data, metadata={"group_access": {50: [1]}}
```

Comment:
Reference ENROLLMENT_TRACK_PARTITION_ID to make "50" less magical. Unfortunately "1" will need to remain magical.

Also please change the other group access access ints to be based off of MINIMUM_STATIC_PARTITION_ID (group IDs should be unique across all partitions).